### DISCUSSION TITLE
What should we call our component library?

### DISCUSSION THREAD

**Discussion by seancolsen** [mathesar-foundation/mathesar#833] [Developer Q&A]
Eventually we'll spin our components library off into its own project separate from Mathesar. At that point it will need its own name, so I think it's worth starting to consider what we might want to call it.

Some things to keep in mind:

- We'll also need a package name on [npm](https://www.npmjs.com/). The package name could potentially be prefixed with `svelte-` or `@centerofci/` or `@mathesar/`. Or it could even be suffixed with `-components`, or `-ui`. It's nice to have the repo name on GitHub match the package name on npm, but not required.
- We should avoid names that would easily be confused with any especially popular projects on npm, even without a prefix/suffix.

**Comment by seancolsen** [comment]
I think it would be neat to stick with the theme of movie characters. There are so many to choose from!

Here are some ideas:

- Morpheus - from The Matrix
- Ambrosius - from Labyrinth ([_zero results on npm!_](https://www.npmjs.com/search?q=ambrosius))
- Falcor (or Falkor) - from The Neverending Story

**Reply by kgodey** [reply]
We could also do other _Galaxy Quest_ characters if we wanted to make that a theme.

**Reply by seancolsen** [reply]
Want to suggest one? I've never seen the movie. Will need to watch it now!

**Reply by kgodey** [reply]
You should definitely watch it!

I'd suggest `Laliari`, she's another Thermian character.

**Comment by kgodey** [comment]
We could also call it something like `mathesar-ui`, it might bring more attention to the main Mathesar project.
